scout/server/blueprints/clinvar/templates/clinvar/clinvar_submissions.html

```diff
@@ -101,9 +100,13 @@ <h2 class="accordion-header" id="header-{{subm_obj._id}}">
               {% endif %}
               {% if subm_obj.status == 'submitted' %} <!--Submission status query -->
                  <form id="statusApi_{{subm_obj._id}}" action="{{ url_for('clinvar.clinvar_submission_status', submission_id=subm_obj.clinvar_subm_id) }}" method="POST">
-                   {{ status_modal(subm_obj._id) }}
+                   {{ action_modal("status", subm_obj._id) }}
```

Comment:
Renamed this modal that provides the ClinVar API key. This so it's used both when asking ClinVar API for the status of a submission and when deleting a submission object